### ISSUE TITLE
Some questions about the verbalizer

### ISSUE THREAD

**Issue by NEMOlv** [thunlp/OpenPrompt#191]
Hi, thank you very much for your open source. I have a few small questions that I hope you can answer.
What is the logic for writing the process_logits() function and the Project() function in manual_verbalizer.py, please? 
Is there a paper on why it is feasible to do the mapping this way? 
Why don't the logits here just use the logits of mlm and formalise the loss as the loss of the mlm task.

May I ask how the following scenario was arrived at? Are there any relevant papers available for reference?
A whole framework to process the original logits over the vocabulary, which contains four steps:
   (1) Project the logits into logits of label words
   if self.post_log_softmax is True:
       (2) Normalize over all label words
       (3) Calibrate (optional)
   (4) Aggregate (for multiple label words)

**Comment by Achazwl** [comment]
For (1)(2), for example: Section 3.1 in "Exploiting Cloze Questions for Few Shot Text Classification and Natural
Language Inference"; Equation 1 in "It’s Not Just Size That Matters:
Small Language Models Are Also Few-Shot Learners"; Equation 1 in "AUTOPROMPT: Eliciting Knowledge from Language Models
with Automatically Generated Prompts"

For (3)(4), for example: "Calibrate Before Use:
Improving Few-Shot Performance of Language Models", "Autoprompt: Eliciting knowledge from language models with automatically generated prompts", "Knowledgeable Prompt-tuning: Incorporating Knowledge into Prompt Verbalizer for Text Classification"
